presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```diff
@@ -609,6 +609,15 @@ public long getBytes()
             return blocked;
         }
 
+        @Override
+        public ListenableFuture<?> setBytes(long bytes, boolean enforceBroadcastMemoryLimit)
+        {
+            ListenableFuture<?> blocked = delegate.setBytes(bytes, enforceBroadcastMemoryLimit);
```

Comment:
have setBytes(long Bytes) call setBytes(bytes, false) so that the logic of setting bytes isn't duplicated.  same for trySetBytes